pybaseball/statcast_pitcher_spin.py

```diff
@@ -19,6 +19,29 @@
 
 
 def statcast_pitcher_spin(start_dt=None, end_dt=None, player_id=None):
+    """
+    Retrieves pitch-level statcast data for a given date or range or dates and calculates spin related metrics.
+
+    ARGUMENTS
+        start_dt: first day for which you want to retrieve data. Defaults to yesterday's date if nothing is entered. 
+            If you only want data for one date, supply a start_dt value but not an end_dt value. Format: YYYY-MM-DD.
+        end_dt: last day for which you want to retrieve data. Defaults to None. If you want to retrieve data for more 
+            than one day, both a start_dt and end_dt value must be given. Format: YYYY-MM-DD.
+        player_id: MLBAM player ID for the pitcher you want to retrieve data for. To find a player's MLBAM ID, 
+            see the function playerid_lookup.
+    
+    RETURNS
+        A dataframe containing pitch-level statcast data for the given dates, along with these colummns added:
+        
+        Mx: The amount of movement in the x-direction due to the Magnus effect alone. (Positive is towards first 
```

Comment:
Maybe this is a lot to have in the code? I could see either way, what do you think? This is another place we could say "see the docs for the added columns."

Reply:
I agree it is a lot-- I just changed it to a bit of an in between, where I indicate that the other columns exist but leave the descriptions of them to the docs.